crypto/bn/bn_print.c

```diff
@@ -67,9 +67,10 @@ char *BN_bn2dec(const BIGNUM *a)
      */
     i = BN_num_bits(a) * 3;
     num = (i / 10 + i / 1000 + 1) + 1;
+    tbytes = num + 3;
```

Comment:
This was in the original code in the malloc which was comment free.

Negative sign and null terminator are two.  I've no idea why 3.
The derivation of the bound on num in the comment above looks correct.